src/test/java/be/seeseemelk/mockbukkit/UnsafeValuesTest.java

```diff
@@ -157,4 +157,28 @@ void fromLegacy_MaterialData_Null_ReturnsNull()
 		assertThrows(NullPointerException.class, () -> mockUnsafeValues.fromLegacy((MaterialData) null));
 	}
 
+	@Test
+	void testBlockTranslationKey()
+	{
+
```

Comment:
About tests, as you now are able to fetch the data from a real server instance, you can simply just make simple tests on ordinary cases, and some edgecases. So to say, assert the translatable from a certain Namespacedkey is equal to hardcoded values like this:
```java
assertEquals("item.minecraft.mockitem", itemStack.getTranslationKey());
```